frontend/src/components/PackageGeneration/Editor.js

```diff
@@ -13,6 +13,17 @@ const code = `a {
     margin:10px;
 }`;
 
+function saveData(blob, fileName) {
```

Comment:
I think you should add Javadocs through this file.  Just to keep everything in line.